tests/cli_integration.rs

```diff
@@ -188,6 +188,7 @@ mod cli_integration {
 	async fn with_root_auth() {
 		// Commands with credentials when auth is enabled, should succeed
 		let (addr, _server) = common::start_server_with_defaults().await.unwrap();
+		println!("addr: {}", addr);
```

Comment:
let's use `debug!` or `trace!` if you want to keep this around, so it's opt-in

Reply:
```suggestion
```